src/components/Tokens/TokenDetails/Skeleton.tsx

```diff
@@ -128,7 +128,7 @@ function Wave() {
   )
 }
 
-function LoadingChart() {
+export function LoadingChart() {
```

Comment:
This file structure is arguably wrong (but this is tech debt).

Ideally, I'd like to see the skeletons live in a file in the same folder as the component that they reflect. Eg, for `ChartSection`, make `ChartSection` a folder and then have `ChartSection/Skeleton.tsx`.

I wouldn't change this in this PR, but it's something to keep in mind - generally, I think the Skeleton should live in the same folder as the component that it reflects.

Reply:
sounds good